### ISSUE TITLE
iOS SubscribedHandler not called

### ISSUE THREAD

**Issue by lauriskr** [cleverpush/cleverpush-flutter-sdk#12]
Hello

I'm trying to get the latest SDK version 1.8.1 to work on iOS, but the subscription ID is not sent via the setSubscribedHandler callback. I tried with an actual project and also with the example project referencing the 1.8.1 library version.

Push notifications are received when a test notification is sent from the web console.

flutter --version:

```
Flutter 2.2.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision d79295af24 (9 weeks ago) • 2021-06-11 08:56:01 -0700
Engine • revision 91c9fc8fe0
Tools • Dart 2.13.3
```

Logs from unsubscribe & subscribe button clicks in the example project:
```
CleverPush: HTTP -> POST https://api.cleverpush.com/subscription/unsubscribe
CleverPush: unsubscribe success
flutter: Prompting for Permission
CleverPush: Device Registered with Apple: 292842d1851bd3c3dc0878239084a928b23a5d089a76016b38f4f894092a8f08
CleverPush: syncSubscription called from registerDeviceToken
CleverPush: syncSubscription called from subscribe
CleverPush: syncSubscription Request data:{
    apnsToken = 292842d1851bd3c3dc0878239084a928b23a5d089a76016b38f4f894092a8f08;
    appVersion = "1.0";
    browserType = SDK;
    browserVersion = "1.12.0";
    country = EE;
    language = en;
    platformName = iOS;
    platformVersion = "14.7.1";
    timezone = "Europe/Tallinn";
    topics =     (
    );
    topicsVersion = 1;
} id:(null)
CleverPush: HTTP -> POST https://api.cleverpush.com/subscription/sync/hxWyS7jPk4DrnSk5K
CleverPush: showing pending topics dialog
CleverPush: syncSubscription Request data:{
    apnsToken = 292842d1851bd3c3dc0878239084a928b23a5d089a76016b38f4f894092a8f08;
    appVersion = "1.0";
    browserType = SDK;
    browserVersion = "1.12.0";
    country = EE;
    language = en;
    platformName = iOS;
    platformVersion = "14.7.1";
    subscriptionId = kyqqi7sTByrZ4xGrs;
    timezone = "Europe/Tallinn";
    topics =     (
        Wdj8TArZi5PsruyxC,
        tabEhawqMfDsvscLn
    );
    topicsVersion = 1;
} id:kyqqi7sTByrZ4xGrs
CleverPush: HTTP -> POST https://api.cleverpush.com/subscription/sync/hxWyS7jPk4DrnSk5K
CleverPush: Device Registered with Apple: 292842d1851bd3c3dc0878239084a928b23a5d089a76016b38f4f894092a8f08
CleverPush: Device Registered with CleverPush: kyqqi7sTByrZ4xGrs
CleverPush: syncSubscription Request data:{
    apnsToken = 292842d1851bd3c3dc0878239084a928b23a5d089a76016b38f4f894092a8f08;
    appVersion = "1.0";
    browserType = SDK;
    browserVersion = "1.12.0";
    country = EE;
    language = en;
    platformName = iOS;
    platformVersion = "14.7.1";
    subscriptionId = kyqqi7sTByrZ4xGrs;
    timezone = "Europe/Tallinn";
    topics =     (
        Wdj8TArZi5PsruyxC,
        tabEhawqMfDsvscLn
    );
    topicsVersion = 1;
} id:kyqqi7sTByrZ4xGrs
CleverPush: HTTP -> POST https://api.cleverpush.com/subscription/sync/hxWyS7jPk4DrnSk5
```

**Comment by gbhrdt** [comment]
Hi @lauriskr thanks for opening this issue.
We were able to replicate it on our side, I will update you asap when a fix is ready.

**Comment by gbhrdt** [comment]
Thanks for your patience. This should now be fixed in version 1.8.2.
